### DISCUSSION TITLE
Enterprise Metadata Bundle

### DISCUSSION THREAD

**Discussion by freela-rag** [pimcore/pimcore#8675] [Q&A / Support]
Hi,

I am trying to install Portal Engine based on https://pimcore.com/docs/portal-engine/current/Installation/Installation.html.

When I try to install Enterprise Metadata Bundle as described in https://pimcore.com/docs/asset-metadata/current/Installation.html (composer require pimcore/asset-metadata-class-definitions) I get the following error:

 [InvalidArgumentException] Could not find a matching version of package pimcore/asset-metadata-class-definitions. Check the package spelling, your version constraint and that the package is available in a s tability which matches your minimum-stability (stable). 

How do I fix this?

Thanks,
Freela

**Comment by fashxp** [comment]
Hi,
since these are enterprise bundles, you need to have access to the enterprise package management of Pimcore. 
Do you have that? 

BR
Christian

**Comment by Fredrik1972** [comment]
Hi!

I have the same issue trying to install Enterprise Metadata Bundle, what is enterprise package management and how do i get it?

**Reply by fashxp** [reply]
These packages are part of our enterprise subscription, see https://pimcore.com/en/platform/subscription for more information.
